[INFO   ] Logger: Record log in kivy_home\logs\kivy_20-08-04_10.txt
[INFO   ] deps: Successfully imported "kivy_deps.gstreamer" 0.1.18
[INFO   ] deps: Successfully imported "kivy_deps.glew" 0.1.12
[INFO   ] deps: Successfully imported "kivy_deps.sdl2" 0.1.23
[INFO   ] Kivy: v2.0.0rc1, git-9398c8d, 20200305
[INFO   ] Kivy: Installed at "C:\Python38-32\lib\site-packages\kivy\__init__.py"
[INFO   ] Python: v3.8.1 (tags/v3.8.1:1b293b6, Dec 18 2019, 22:39:24) [MSC v.1916 32 bit (Intel)]
[INFO   ] Python: Interpreter at "C:\Python38-32\python.exe"
[INFO   ] Factory: 184 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2 (img_pil, img_ffpyplayer, img_gif ignored)
[INFO   ] Text: Provider: sdl2
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "main.py", line 1316, in <module>
[WARNING] stderr:     PlannerApp().run()
[WARNING] stderr:   File "C:\Python38-32\lib\site-packages\kivy\app.py", line 949, in run
[WARNING] stderr:     self._run_prepare()
[WARNING] stderr:   File "C:\Python38-32\lib\site-packages\kivy\app.py", line 918, in _run_prepare
[WARNING] stderr:     self.load_kv(filename=self.kv_file)
[WARNING] stderr:   File "C:\Python38-32\lib\site-packages\kivy\app.py", line 691, in load_kv
[WARNING] stderr:     root = Builder.load_file(rfilename)
[WARNING] stderr:   File "C:\Python38-32\lib\site-packages\kivy\lang\builder.py", line 304, in load_file
[WARNING] stderr:     return self.load_string(data, **kwargs)
[WARNING] stderr:   File "C:\Python38-32\lib\site-packages\kivy\lang\builder.py", line 371, in load_string
[WARNING] stderr:     parser = Parser(content=string, filename=fn)
[WARNING] stderr:   File "C:\Python38-32\lib\site-packages\kivy\lang\parser.py", line 402, in __init__
[WARNING] stderr:     self.parse(content)
[WARNING] stderr:   File "C:\Python38-32\lib\site-packages\kivy\lang\parser.py", line 511, in parse
[WARNING] stderr:     objects, remaining_lines = self.parse_level(0, lines)
[WARNING] stderr:   File "C:\Python38-32\lib\site-packages\kivy\lang\parser.py", line 614, in parse_level
[WARNING] stderr:     _objects, _lines = self.parse_level(
[WARNING] stderr:   File "C:\Python38-32\lib\site-packages\kivy\lang\parser.py", line 614, in parse_level
[WARNING] stderr:     _objects, _lines = self.parse_level(
[WARNING] stderr:   File "C:\Python38-32\lib\site-packages\kivy\lang\parser.py", line 614, in parse_level
[WARNING] stderr:     _objects, _lines = self.parse_level(
[WARNING] stderr:   [Previous line repeated 5 more times]
[WARNING] stderr:   File "C:\Python38-32\lib\site-packages\kivy\lang\parser.py", line 674, in parse_level
[WARNING] stderr:     if current_property[:3] == 'on_':
[WARNING] stderr: TypeError: 'NoneType' object is not subscriptable
